flood.py

```diff
@@ -29,6 +29,13 @@ def gen_json(hypervisors, guests):
 
 def rm_container(client, containers, reason="Success"):
     del_container = containers[0]
+    container_log = open("container.log", "a")
```

Comment:
Just curious why you don't use a context manager to work with the file.

Reply:
only reason is that I havent coded in python in a while ;) Updating now